spec/support/test_helpers/candidate_helper.rb

```diff
@@ -27,6 +27,7 @@ def candidate_completes_application_form(with_referees: true)
     FeatureFlag.deactivate(:international_personal_details)
     FeatureFlag.deactivate(:efl_section)
     FeatureFlag.deactivate(:international_degrees)
+    FeatureFlag.deactivate(:international_other_qualifications)
```

Comment:
Is this acceptable? If this feature flag is active, then in the tests, for the "other" qualification, we get a blank string as the `non_uk_qualification_type` rather than `nil` in the API response